webapp/src/shared/components/Row/Row.css

```diff
@@ -0,0 +1,26 @@
+.row {
+  display: flex;
+  width: 100%;
+  min-width: fit-content;
+  height: 82px;
+  padding: 16px 28px;
+  border-radius: 8px;
```

Comment:
I think we could use `rem` units better, this component may be bigger in desktop and, using this units, it would scale automatically depending on base font size.